backend/src/relationships/blocks/entities/block.entity.ts

```diff
@@ -10,34 +10,35 @@ import {
   Unique,
 } from 'typeorm';
 
-export enum RelationshipTypeEnum {
-  FRIEND = 'friend',
-  BLOCK = 'block',
+export enum BlockTypeEnum {
+  T_BLOCKS_S = 1,
+  S_BLOCKS_T = 2,
+  MUTUAL = 3,
```

Comment:
```suggestion
  T_BLOCKS_S = "target_blocks_source",
  S_BLOCKS_T = "source_blocks_target",
  MUTUAL = "mutual",
```
Same question as below with the friend Entity

Reply:
changed in next commit